bin/lerna.js

```diff
@@ -34,6 +34,7 @@ var cli = meow([
   "  --repo-version       Specify repo version to publish",
   "  --concurrency        How many threads to use if lerna parallelises the tasks (defaults to 4)",
   "  --loglevel           What level of logs to report (defaults to \"info\").  On failure, all logs are written to lerna-debug.log in the current working directory.",
+  "  --quiet              Don't show current lerna version as first line of output. Useful for `lerna ls`",
```

Comment:
Probably a good idea. First PR actually. Can't seem to change the branch the PR is made from. I'll make a PR without the --quiet flag.